lua/cmake/utils.lua

```diff
@@ -89,7 +90,18 @@ function utils.get_current_target(parameters)
     return nil, nil
   end
 
-  local target_dir = vim.fn.fnamemodify(target, ':h')
+  local target_dir
+  if parameters['runDir'] ~= nil then
+    local run_path = Path:new(parameters['runDir'])
+    if run_path:is_absolute() then
+      target_dir = run_path:normalize(vim.loop.cwd())
```

Comment:
We do we need to call normalize if the path is absolute?

Reply:
That is true. It isn't needed. I will clean this section up